vtex-apps/lightspeed
# [LIGHTSpD-37] add shipping

**What problem is this solving?**

Added shipping information in CreateSaleLine step

**How should this be manually tested?**

[https://aurora--benscycle.myvtex.com/admin/lightspeed](url)
Retry an order in the workspace and go to the order detail and you should see the `shipPrice` and update amout

ex : [https://api.lightspeedapp.com/API/Account/4767/Sale/1037355.json?load_relations=["SalePayments.PaymentType", "SaleLines"]](url)

[https://benscycle.myvtex.com/lightspeed/_v/0/check/BC540246-01](url)
